eng/common/testproxy/test-proxy-tool.yml

```diff
@@ -37,11 +31,7 @@ steps:
 
   # nohup does NOT continue beyond the current session if you use it within powershell
   - bash: |
-      sudo nohup $(Build.BinariesDirectory)/test-proxy/test-proxy &
+      nohup $(Build.BinariesDirectory)/test-proxy/test-proxy &
```

Comment:
So it turned out sudo is not needed on linux any longer?

Reply:
Nope! I pulled a

> let's just confirm that

And removed it, and linux didn't have any issues.

I have a bad feeling it's related to writing the output to a file, but that's a problem for tomorrow scott. Need to get this out ASAP such that PRs can start working properly 👍